decidim-elections/config/locales/en.yml

```diff
@@ -23,10 +30,24 @@ en:
         title: Actions
         view_deleted_elections: View deleted elections
       admin:
+        census:
+          edit:
+            choose_census: Choose the type of census you want to use for this election
+            created_at: Created at
+            identifier: User identifier
+            title: Type of census
+            voters_count: There are currently %{count} people eligible for voting in this election (this might change on a dynamic census).
+          preview:
+            title: 'User preview (the list is limited to %{limit} registers):'
```

Comment:
i am not sure what "registers" means here: Is the plural of "registry" ( if so the text is correct), or is "registration" / "register" ( enrollment ) which means that you may need to revisit the text, or even add a plural form.

Reply:
@antopalidi @alecslupu yeah, "registers" is a mistranslation from Spanish. Let's stick with "records" since that's the term we're already using in the Organization Census section:
![image](https://github.com/user-attachments/assets/ee0fb381-c489-4287-93e4-f1679b6c4e27)
Thanks for catching that!